drivers/gles3/shaders/blend_shape.glsl

```diff
@@ -103,7 +103,7 @@ out vec2 uv2_out; //tfb:ENABLE_UV2
 #endif
 
 #ifdef ENABLE_SKELETON
-out ivec4 bone_out; //tfb:ENABLE_SKELETON
+flat out ivec4 bone_out; //tfb:ENABLE_SKELETON
```

Comment:
See https://github.com/lyuma/godot/commit/3ee3048b1e6fb961512bedb9cc13c2f73d606228 for details.

Reply:
Were gles_builders.py changes added?

Reply:
They should have been, I'll add them back.

Reply:
I can't seem to fin them in your groups-angle branch, can you link me to them?

Reply:
https://github.com/lyuma/godot/commit/3ee3048b1e6fb961512bedb9cc13c2f73d606228